PhoneBook/src/main/java/humphrey/jeriah/PhoneBook.java

```diff
@@ -0,0 +1,52 @@
+package humphrey.jeriah;
+import java.util.*;
+
+/**
+ * Created by jeriahhumphrey on 2/7/17.
+ */
+public class PhoneBook {
+
+     TreeMap <String, String> phoneBook = new TreeMap();
+
+
+     public String lookUp(String key){
+         return phoneBook.get(key);
+     }
+
+     public void addEntry (String name, String number){
+         phoneBook.put(name, number);
+
+     }
+
+     public TreeMap getMap(){
```

Comment:
What's the purpose of `getMap()`? Is it only for testing purposes? As its implemented this method allows direct access to the phonebook which can result in unintentional modification. It also breaks encapsulation a little if you intend to use it for things like seeing how many entries are in the phonebook.